src/AnimatedValue.js

```diff
@@ -10,13 +10,16 @@
  */
 'use strict';
 
-var AnimatedWithChildren = require('./AnimatedWithChildren');
-var InteractionManager = require('./injectable/InteractionManager');
-var AnimatedInterpolation = require('./AnimatedInterpolation');
-var Interpolation = require('./Interpolation');
-var Animation = require('./Animation');
-var guid = require('./guid');
-var Set = global.Set || require('./SetPolyfill');
+import AnimatedWithChildren from './AnimatedWithChildren';
+import InteractionManager from './injectable/InteractionManager';
+import AnimatedInterpolation from './AnimatedInterpolation';
+import Interpolation from './Interpolation';
+import Animation from './Animation';
+import guid from './guid';
+import SetPilyfill from './SetPolyfill';
+
+// TODO: wonder if we should do the set polyfill another way...
```

Comment:
It would be nice if the `Set` polyfill from es6-shim was a standalone module that you could just depend on. Alternatively, you could put the burden on consumers and make shimming `Set` a requirement.

@ljharb might have some thoughts about how to do this best.

Reply:
I think it’s way simpler for a standard polyfillable feature like Set (or Map, or Promise, etc) to just assume it’s present, and state in the docs that a polyfill is needed.